src/views/DownloadPage.vue

```diff
@@ -48,28 +47,17 @@ const fetchBuilds = async () => {
           windowBuildUrl.value = asset.browser_download_url;
         } else if (asset.name.endsWith("linux_x64.tar.gz")) {
           linuxBuildUrl.value = asset.browser_download_url;
+        } 
+      } else if (asset.name.startsWith("test-ava-ryujinx")) {
+        if (asset.name.endsWith(".app.tar.gz")) {
```

Comment:
```suggestion
        if (asset.name.endsWith("macos_universal.app.tar.gz ")) {
```
Just for consistency with the other cases here.